step-core/src/main/java/step/automation/packages/AutomationPackageOperationMode.java

```diff
@@ -0,0 +1,7 @@
+package step.automation.packages;
+
+public enum AutomationPackageOperationMode {
+    MAIN,
```

Comment:
I would add a javadoc to document these 3 modes. Not Sure about the meaning of "MAIN"...

Reply:
Hehe, yes! "MAIN" was just a fallback name because I didn't find a better one. The actual meaning is something like "PRODUCTION", or "REALLY_DO_IT" (as opposed to the other two), but these seemed even less appropriate, so I tried to use something generic. Any suggestion for the correct name? :)